pkg/schema/tracker_test.go

```diff
@@ -487,3 +487,25 @@ func (s *trackerSuite) TestAllSchemas(c *C) {
 	_, err = tracker.GetTable("testdb2", "a")
 	c.Assert(err, ErrorMatches, `.*Table 'testdb2\.a' doesn't exist`)
 }
+
+func (s *trackerSuite) TestNotSupportedVariable(c *C) {
+	log.SetLevel(zapcore.ErrorLevel)
+
+	db, mock, err := sqlmock.New()
+	c.Assert(err, IsNil)
+	defer db.Close()
+	con, err := db.Conn(context.Background())
+	c.Assert(err, IsNil)
+	baseConn := conn.NewBaseConn(con, nil)
+
+	mock.ExpectQuery("SHOW VARIABLES LIKE 'sql_mode'").WillReturnRows(
+		sqlmock.NewRows([]string{"Variable_name", "Value"}).
+			AddRow("sql_mode", ""))
+
+	oldSessionVar := map[string]string{
+		"tidb_enable_change_column_type": "ON",
+	}
+
+	_, err = NewTracker(context.Background(), "test-tracker", oldSessionVar, baseConn)
+	c.Assert(err, IsNil)
```

Comment:
in fact this err should not be nil. And if downstream TiDB version is different with schema tracker, user may set `session` in config for the purpose of setting downstream session variables, and not expected schema tarcker raising an error. 🤔

Reply:
yes, i try igonre this error and give a warning msg https://github.com/pingcap/dm/pull/1910/commits/34767bc89c210f5cd1a7beca959ecd29ac758fc5